__tests__/app.test.js

```diff
@@ -1,40 +1,115 @@
-const app = require('../app');
-const runSeed = require('../db/seeds/seed');
-const testData = require('../db/data/test-data');
-const request = require('supertest');
+const app = require("../app");
+const runSeed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+const request = require("supertest");
 beforeEach(() => runSeed(testData));
-describe('GET non-existent endpoint', () => {
-    test("returns 404", () => {
-        return request(app)
-        .get('/api/cagetories')
-        .expect(404);
-    });
-    test("returns error message", () => {
-        return request(app)
-        .get('/api/test')
-        .then((res) => {
-            expect(res.body).toEqual({ msg: "error: invalid path" });
-        })
-    })
-})
-describe('GET /api/categories', () => {
-    test("returns status code 200", () => {
-        return request(app)
-        .get('/api/categories')
-        .expect(200);
-    });
-    test("returns array of objects", () => {
-        return request(app)
-        .get('/api/categories')
-        .then((res) => {
-            expect(Array.isArray(res.body.categories)).toBe(true);
-        })
-    })
-    test("returns correct response", () => {
-        return request(app)
-        .get('/api/categories')
-        .then((res) => {
-            expect(res.body.categories).toMatchObject(testData.categoryData);
-        })
-    })
-})
\ No newline at end of file
+describe("GET non-existent endpoint", () => {
+  test("returns 404", () => {
+    return request(app).get("/api/cagetories").expect(404);
+  });
+  test("returns error message", () => {
+    return request(app)
+      .get("/api/test")
+      .then((res) => {
+        expect(res.body).toEqual({ msg: "error: invalid path" });
+      });
+  });
+});
+describe("GET /api/categories", () => {
+  test("returns status code 200", () => {
+    return request(app).get("/api/categories").expect(200);
+  });
+  test("returns array of objects", () => {
+    return request(app)
+      .get("/api/categories")
+      .then((res) => {
+        expect(Array.isArray(res.body.categories)).toBe(true);
+      });
+  });
+  test("returns correct response", () => {
+    return request(app)
+      .get("/api/categories")
+      .then((res) => {
+        expect(res.body.categories).toMatchObject(testData.categoryData);
```

Comment:
This will work for this one, but not future endpoints, as the testData wont always look like the testData in the database. 
Ideally, you want to forEach through the array, then check each .toMatchObject - looks like what you expect .e.g. `.toMatchObject({keyHere: expect.any(String), KeyHere:expect.any(Number)})`etc

Reply:
i'd leave it for now and try get moving on, but be aware you'll have to do it the way suggested above for future tests, or it wont pass